### DISCUSSION TITLE
Remove 'undo commit' command from gitlens inspect?

### DISCUSSION THREAD

**Discussion by mdj-uk** [gitkraken/vscode-gitlens#2745] [Q&A] [answered]
When in the gitlens inspect file history, or commit history, there is a button to undo the latest commit (the first icon from the left):
<img width="305" alt="image" src="https://github.com/gitkraken/vscode-gitlens/assets/101579982/7b829ebd-cf0f-45c1-9dbb-fc99de4e379d">

I find it very easy to hit that accidentally, which makes me scared to use just about any gitlens feature. 

Is there any way to remove it, or to require a confirmation step? I couldn't find any option in the extension settings.

**Comment by jkelroy** [comment, marked as answer]
Hello @mdj-uk, this is not currently possible but a feature request to support this has been opened here #2746.

As a note, this will only do a soft reset so it is not a destructive action.

**Reply by mdj-uk** [reply]
Thanks, will follow the issue
